src/stemmablenote.ts

```diff
@@ -58,13 +57,13 @@ export abstract class StemmableNote extends Note {
 
     if (this.hasFlag()) {
       const flagCode =
-        this.getStemDirection() === Stem.DOWN ? glyphProps.codeFlagDownstem ?? '' : glyphProps.codeFlagUpstem ?? '';
+        // codeFlagDown = codeFlagUp + 1,, if not defined, code should be 0
+        this.getStemDirection() === Stem.DOWN
+          ? String.fromCodePoint((glyphProps.codeFlagUp?.codePointAt(0) ?? -1) + 1)
+          : glyphProps.codeFlagUp ?? '\u0000';
```

Comment:
@rvilarl I'm going to merge, but this block could still be improved, I think. :-)